cloudify_agent/api/pm/base.py

```diff
@@ -15,6 +15,8 @@
 
 import getpass
 import os
+from pkg_resources import resource_filename
```

Comment:
`pkg_resources` is 3rd party, it should come after the `from celery import Celery` import